projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml

```diff
@@ -59,6 +59,35 @@ frontend-tests-vdk-jupyterlab-extension:
     reports:
       junit: "projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/frontend_tests_results.xml"
 
+integration-tests-vdk-jupyterlab-extension:
+  image: "mcr.microsoft.com/playwright:v1.34.0-jammy"
+  # extends: .common-setup-vdk-jupyterlab-extension
+  stage: pre_release_test
+  script:
+    - export TZ=Europe/Minsk
+    - ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
Why do we need to set the timezone to ours? AFAIK, we don't do that in other CI files. I think it might potentially cause some unwanted time discrepancies in the future.